packages/rich-text/src/create.js

```diff
@@ -178,6 +178,19 @@ export class RichTextData {
 	}
 }
 
+for ( const name of Object.getOwnPropertyNames( String.prototype ) ) {
+	if ( RichTextData.prototype.hasOwnProperty( name ) ) {
+		continue;
+	}
+
+	Object.defineProperty( RichTextData.prototype, name, {
+		value( ...args ) {
+			// Should we convert back to RichTextData?
+			return this.toHTMLString()[ name ]( ...args );
```

Comment:
I think it would be great if we convert back to RichTextData here, but I'm not sure. One downside would be that you'd be parsing and serialising every time you use one of these methods.

Reply:
In the future, should also warn when using these methods, but not until we have alternative methods available.